CONTRIBUTING.md

```diff
@@ -412,10 +412,6 @@ Being in this team will grant you the following access:
   community moderators.  You should hide them instead.  An audit log
   is available to track deleted items if needed (but does not allow
   recovering them).
-- **Pushing a branch or a tag to the main repository:** please push
-  changes to your own fork rather than the main repository. (Branches
-  pushed to the main repository will be removed promptly and without
-  notice.)
```

Comment:
Shouldn't we say "pushing branches to the main repo is disabled" rather than say nothing?